netmanager-app/lib/utils.ts

```
@@ -8,3 +9,36 @@ export function cn(...inputs: ClassValue[]) {
+    type: "Feature";
+    properties: unknown;
+    geometry: { type: "Point"; coordinates: [number, number] };
+  }[] = [];
+  data.map((feature) => {
+    if (filter(feature)) {
+      features.push({
+        type: "Feature",
+        properties: { ...rest, ...feature },
+        geometry: {
+          type: "Point",
+          coordinates: (coordinateGetter && coordinateGetter(feature)) || [
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)